docs/ui.go

```diff
@@ -39,10 +39,15 @@ var ui = `
     <script src="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/3.20.5/swagger-ui-bundle.js"> </script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/3.20.5/swagger-ui-standalone-preset.js"> </script>
     <script>
+    function getBaseURL() {
+      var path = window.location.pathname
+      path = path.substr(0, path.lastIndexOf('/')+1)
+      return window.location.host + path;
+    }
     window.onload = function() {
       // Begin Swagger UI call region
       const ui = SwaggerUIBundle({
-        url: "../swagger",
```

Comment:
The relative URL from `/docs` to `/swagger` should be `swagger`, not `../swagger`. This is fixed automatically by the browser if it is served at root but that was not the case with a path prefix.